bcda/web/api.go

```diff
@@ -31,12 +33,27 @@ import (
 	"github.com/CMSgov/bcda-app/bcda/utils"
 )
 
-var qc *que.Client
+var (
+	qc *que.Client
+)
 
 const (
 	groupAll = "all"
 )
 
+func init() {
+	// Ensure that models.go is properly initialized with the service reference.
+	// As we refactor more of the code, we should be able to remove the initialization
+	// from models.go
+	cutoffDuration := time.Duration(utils.GetEnvInt("CCLF_CUTOFF_DATE_DAYS", 45)*24) * time.Hour
+	db := database.GetGORMDbConnection()
+	db.DB().SetMaxOpenConns(utils.GetEnvInt("BCDA_DB_MAX_OPEN_CONNS", 25))
```

Comment:
Why 25, 25,  and. 5 for these values?

Reply:
I was looking at some posts about perf benefits and came across this one: https://www.alexedwards.net/blog/configuring-sqldb that seemed to match our size of reqeusts.